### PR TITLE
Feature/add ci environment

### PR DESCRIPTION
**Pull request by JuanCamilo08**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:13.13.0
+FROM node:13.13.0-alpine3.10
```
Comment (cesar18pena) [inline Dockerfile]: Never use unstable NodeJS version unless you are testing something

Comment (JuanCamilo08) [reply Dockerfile]: You recommend to use `LTS` versions? or what do you mean by unstable?

Comment (cesar18pena) [reply Dockerfile]: LTS versions are stable and latest versions are unstable

Comment (JuanCamilo08) [reply Dockerfile]: TY!

**Pull request merged**